bindata/bootkube/scc-manifests/0000_20_kube-apiserver-operator_00_crb-systemauthenticated-scc-restricted-v2.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:scc:restricted-v2
```

Comment:
@stlaz this needs the platform annotations, this is currently breaking hyeprshift (the revert didn't get get promoted) and presumably all other non-default releases

/hold

Reply:
@alvaroaleman do you mind to elaborate on the "platform annotations"? what do you we need to set there?

Reply:
The same as cesar, seems we commented at about the same time. Looks good now, thanks!
/hold cancel